tests/Internal/Codebase/InternalCallMapHandlerTest.php

```diff
@@ -378,31 +365,17 @@ class InternalCallMapHandlerTest extends TestCase
         'pdo::sqlitecreateaggregate',
         'pdo::sqlitecreatecollation',
         'pdo::sqlitecreatefunction',
-        'pdostatement::__sleep',
-        'pdostatement::__wakeup',
         'simplexmlelement::__get',
```

Comment:
SimpleXmlElement also implement property access without defining a __get() function. I don't know if psalm needs this to allow arbitrary property access.

Reply:
Psalm does indeed checks for __get/__set method existance at some point